core/build.gradle

```diff
@@ -59,7 +51,7 @@ dependencies {
     api "org.apache.curator:curator-recipes:$curatorVersion"
```

Comment:
same for curator stuff, probably not required in control-plane-core module

Reply:
Will leave curator for now as the worker will need it anyways and removing it here would need to be done in conjunction with adding it in the mantis worker.